packages/services/api/src/modules/cdn/providers/cdn.provider.ts

```diff
@@ -26,7 +26,6 @@ export class CdnProvider {
     logger: Logger,
     private session: Session,
     private auditLog: AuditLogRecorder,
-    private auditLogManager: AuditLogManager,
```

Comment:
No need to inject the `AuditLogManager` for this function, as we can just use the `maskToken` function from `@hive/service-common`.